src/components/SummaryList.scss

```diff
@@ -0,0 +1,42 @@
+.SummaryList-header {
+  color: $base-font-color;
+  .SummaryList-header-description {
+    color: $base-font-color;
+    float: left;
+  }
+  .SummaryList-header-value {
+    color: $base-font-color;
+    float: right;
+    clear: right;
+  }
+}
+
+.SummaryList-row {
```

Comment:
Nit: The name row here feels a little off to me... the list children feel more like the rows... maybe it could be something more generic like `SummaryList-section`?

Reply:
Yeah, good call. Changed to section.